From 4e45d50d676f293e837c27c066ee3cd29403c1ad Mon Sep 17 00:00:00 2001
From: Lena Boeckmann <einhornhool@mobi31.inet.haw-hamburg.de>
Date: Wed, 25 Sep 2019 11:35:22 +0200
Subject: [PATCH] set cflags and correct command

---
 lib/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index 6cba30b..aef823d 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -80,6 +80,7 @@ endif()
 add_library(cryptoauth ${CRYPTOAUTH_SRC} ${ATCACERT_DEF_SRC})
 
 set_property(TARGET cryptoauth PROPERTY C_STANDARD 99)
+set(CMAKE_C_FLAGS "${CFLAGS}")
 
 # Add Remaining Sources depending on target library type
 if(ATCA_PKCS11)
@@ -90,7 +91,7 @@ if(ATCA_PRINTF)
 add_definitions(-DATCAPRINTF)
 endif(ATCA_PRINTF)
 
-include_directories(cryptoauth PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ../third_party/hidapi/hidapi ${LIBUSB_INCLUDE_DIR})
+target_include_directories(cryptoauth PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ../third_party/hidapi/hidapi ${LIBUSB_INCLUDE_DIR})
 
 if(WIN32)
 set_target_properties(cryptoauth PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS true)
-- 
2.21.0

